draft-pauly-add-resolver-discovery.md

```diff
@@ -146,7 +150,38 @@ See {{local-deployment}} for local deployment considerations.
 
 # Discovery of DoH Capabilities for Direct Resolvers
 
-- Companion DoH server
+Direct Resolvers can advertise a Companion DoH server that offers equivalent services and is controlled 
+by the same entity. To do this, a DNS server MUST return an HTTPSSVC record for the "resolver.arpa"
```

Comment:
We may just want to say "a DNS server returns", and not have a MUST. It's more a statement of fact than a normative requirement.

Reply:
That's a good point, I'll revise that accordingly.

Reply:
Done